CMakeLists.txt

```diff
@@ -19,30 +19,51 @@ INCLUDE(CMakeDetectOsXArchs)
 PROJECT(s25client)
 
 IF (DEFINED CMAKE_TOOLCHAIN_FILE)
-	MESSAGE(STATUS "Using Toolchain definition file '${CMAKE_TOOLCHAIN_FILE}'")
+	MESSAGE(STATUS "Used Toolchain definition file '${CMAKE_TOOLCHAIN_FILE}'")
 ENDIF()
 
 STRING(TOLOWER ${CMAKE_SYSTEM_NAME} PLATFORM_NAME)
 SET(PLATFORM_NAME ${PLATFORM_NAME} CACHE STRING "Do not change" FORCE)
-SET(PLATFORM_ARCH ${CMAKE_SYSTEM_PROCESSOR} CACHE STRING "Do not change" FORCE)
+
+#Set PLATFORM_ARCH
+IF(${CMAKE_SYSTEM_NAME} STREQUAL "Windows")
```

Comment:
Weil wir wissen wollen, FÜR was compiliert wird. CMake liefert aber immer die architekture des hosts in dem lokalen Fall -.- War vor den Änderungen auch aber hatte gehofft, dass das nicht mehr nötig ist